README.md

```diff
@@ -1,2 +1,5 @@
 # storr.remote
-Interface for plugging in remote storage (e.g. AWS S3) to [storr](https://github.com/richfitz/storr)
+
+[![Travis-CI Build Status](https://travis-ci.org/richfitz/storr.remote.svg?branch=master)](https://travis-ci.org/richfitz/storr.remote)
```

Comment:
Thanks for adding the travis stuff - I have a lot to learn about all this stuff!